packages/storybook-react/src/stories/Document/Document.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Document, Heading1, Paragraph } from '@utrecht/component-library-react';
+import React from 'react';
+
+export default {
+  title: 'React.js Component/Document',
+  id: 'react-document',
+  component: Document,
+} as ComponentMeta<typeof Document>;
+
+const Template: ComponentStory<typeof Document> = (args) => <Document {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = {
+  children: (
+    <>
```

Comment:
The fragment results in a code snippet that is not easy to understand, I suggest we use Article instead.